exchange/adapter_util.go

```diff
@@ -81,6 +81,7 @@ func GetDisabledBiddersErrorMessages(infos config.BidderInfos) map[string]string
 		"lifestreet":   `Bidder "lifestreet" is no longer available in Prebid Server. Please update your configuration.`,
 		"adagio":       `Bidder "adagio" is no longer available in Prebid Server. Please update your configuration.`,
 		"somoaudience": `Bidder "somoaudience" is no longer available in Prebid Server. Please update your configuration.`,
+		"yssp":         `Bidder "yssp" is no longer available in Prebid Server. Please update your configuration.`,
```

Comment:
Yahoo is still in PBS under a different name. I suggest we use a different error message here indicating they can change it to yahoossp and be fine.